src/main/java/ticketsproject/service/TicketDataBaseService.java

```diff
@@ -0,0 +1,7 @@
+package ticketsproject.service;
+
+import ticketsproject.model.TicketDataBase;
+
+public interface TicketDataBaseService {
+    TicketDataBase findById(Long id);
```

Comment:
DataBase is that necessary?
in general in entity name))

Reply:
Do you mean to change name of entity ? Can I change it to TicketInfo ? I have already had Ticket and BusTicket entities in my project

Reply:
Yes) and the same for the User.. without this DataBase

Reply:
Ok , thank you very much )